packages/wordcount/src/index.js

```diff
@@ -0,0 +1,193 @@
+/**
+ * Word counting utility
+ *
+ * Counts the number of words (or other specified type) in the specified text.
+ *
+ * @summary  Count the number of elements in a text.
+ *
+ * @since    2.6.0
+ * @memberof wp.utils.wordcounter
```

Comment:
Should we drop this line, I guess it will be different now and I wonder if we'd have a global at all.